components/tooltip/tooltip.js

```diff
@@ -253,8 +253,8 @@ class Tooltip extends RtlMixin(LitElement) {
 
 			.d2l-tooltip-content {
 				background-color: var(--d2l-tooltip-background-color);
-				border-radius: ${contentBorderRadius}px;
 				border: ${contentBorderSize}px solid var(--d2l-tooltip-border-color);
+				border-radius: ${contentBorderRadius}px;
```

Comment:
These variables prevent us from enforcing lower case for property value functions.  If we convert these to lower case then we can enable, but that's gross too.  Some other work-arounds are possible too.